styles/components/_promo.scss

```diff
@@ -7,7 +7,11 @@
 .sprk-c-Promo {
   background-color: $sprk-promo-background-color;
   max-width: $sprk-promo-max-width;
-  margin: 0 auto;
+  margin: $sprk-promo-margin;
+}
```

Comment:
verify we marked promo as breaking in release notes

Reply:
Checked. It's in there.